### ISSUE TITLE
Users can scroll through and interact with storylines

### ISSUE THREAD

**Issue by mattijsstam** [ZEnMo/Holon-webapp#107]
**Description**

To be able to show storylines to the users based in the input from the CMS as developed in #93 several interactive components have to be build to create the [design](https://xd.adobe.com/view/fdfd0445-f79f-426d-a864-c0cb70e3329b-a8ae/screen/2b23d2b5-55ff-4888-a092-b86df8167083)

![Image](https://user-images.githubusercontent.com/32770108/193806860-0dc73e64-0dcb-4332-ae3f-0ccb46d3f6fb.png)

A storyline should be a single page where users can scroll trough sections with the following elements:

- Static introduction block (text image, text video, etc)
- Dynamic block, which renders the following elements:
  * Richtext with sliders and radio buttons on the left side
  * Animation on the right side which is animated based on the active layers, sliders and radio inputs
  * Kpi's (not designed yet)

All input to build this page should be returned from the CMS

For the model configuration we use the model interface data format #91 

**To discuss**

- Should the animation be part of this story or should it be a seperate user story?
- Where are kpi's shown on the page?
- Animation is out of scope Nice to have if it works on the current animation

**Acceptance criteria**

- Users can scroll trough storylines
- Users see the animation change based on their input
- Users can change radio buttons or sliders and see the result in the animation and kpi's

**Comment by mattijsstam** [comment]
Is blocked by #93

**Comment by pascalalferink** [comment]
A start has been made in the branch StoryLine
